warehouse/common/src/test/java/datawave/common/io/FilesTest.java

```diff
@@ -8,19 +8,10 @@
 import static org.easymock.EasyMock.expect;
 import static org.easymock.EasyMock.mock;
 import static org.easymock.EasyMock.replay;
-
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
No star imports, please.

Reply:
Default IDE behavior.  It loves to help.  Changed the settings and reverted back to the 3 imports.